reader/__main__.py

```diff
@@ -47,19 +47,18 @@
 
 # Reader imports
 import reader
-from reader import feed
-from reader import viewer
+from reader import feed, viewer
 
 
-def main():  # type: () -> None
+def main() -> None:
     """Read the Real Python article feed"""
```

Comment:
If we adhere to [PEP 257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings), then this type of docstring should end in a period.